### PR TITLE
Support setting Verbosity of GitVersion

### LINKED ISSUE #955: Introduce Verbose/Debug logging level
We are adding more and more logging in and often I want less info. We should create another logging level and enable it with a --debug switch or a config setting?

### PR DESCRIPTION
**Pull request by tomap**
Hi,

I added support to set verbosity of GitVersion when launching command line

GitVersion.exe -Verbosity Error allows to have less info/debug messages in your logs

fixes #955 

Thomas P.

### CODE REVIEW

File: src/GitVersionTfsTask/GitVersion.ps1 [hunk 1]
```diff
@@ -54,6 +54,13 @@ if($additionalArguments)
   $argsGitVersion = $argsGitVersion + " $additionalArguments"
 }
 
+if(!$additionalArguments -or !$additionalArguments.Contains("-debug"))
+{
+  if([string]::IsNullOrEmpty($env:SYSTEM_DEBUG) -or $env:SYSTEM_DEBUG -ne "true"){
```
Comment (JakeGinnivan) [inline src/GitVersionTfsTask/GitVersion.ps1]: This is too generic for a variable name. We should name it `GITVERSION_LOG_LEVEL`

**Comment by JakeGinnivan** [comment]
The build is fixed now. If you could change the variable above and rebase this branch that would be great. You should be able to just `git pull --rebase` to rebase instead of merge

File: src/GitVersionTfsTask/GitVersion.ps1 (hunk 1 above)
Comment (JakeGinnivan) [inline src/GitVersionTfsTask/GitVersion.ps1]: I see where you were going with this now after looking through the commits. I am going to remove for now. I think we should extend the VSTS task to expose the log level.

**Comment by JakeGinnivan** [comment]
This is in now through #1107. Thanks!

File: src/GitVersionTfsTask/GitVersion.ps1 (hunk 1 above)
Comment (JakeGinnivan) [inline src/GitVersionTfsTask/GitVersion.ps1]: I removed this for the moment. I figured it can be another PR which we can look at separately

Comment (pascalberger) [inline src/GitVersionTfsTask/GitVersion.ps1]: Makes sense
